src/bp-moderation/classes/class-bp-moderation-media.php

```diff
@@ -180,4 +190,33 @@ public function report_button_text( $button_text, $item_id ) {
 	public function report_content_type( $content_type, $item_id ) {
 		return esc_html__( 'Photo', 'buddyboss' );
 	}
+
+	/**
+	 * Function to update activity report button arguments.
+	 *
+	 * @param array $report_button Activity report button
+	 * @param array $args          Arguments
+	 *
+	 * @return array|string
+	 */
```

Comment:
```suggestion
	/**
	 * Function to update activity report button arguments.
	 * 
	 * @since BuddyBoss 1.7.6
	 *
	 * @param array $report_button Activity report button.
	 * @param array $args          Arguments.
	 *
	 * @return array|string
	 */
```